pkg/skaffold/test/test.go

```diff
@@ -74,17 +76,17 @@ func (t FullTester) Test(ctx context.Context, out io.Writer, bRes []build.Artifa
 	return nil
 }
 
-func (t FullTester) runStructureTests(ctx context.Context, out io.Writer, bRes []build.Artifact, testCase *latest.TestCase) error {
-	if len(testCase.StructureTests) == 0 {
+func (t FullTester) runStructureTests(ctx context.Context, out io.Writer, bRes []build.Artifact, tc *latest.TestCase) error {
+	if len(tc.StructureTests) == 0 {
 		return nil
 	}
 
-	files, err := util.ExpandPathsGlob(t.workingDir, testCase.StructureTests)
-	if err != nil {
-		return fmt.Errorf("expanding test file paths: %w", err)
+	fqn, found := resolveArtifactImageTag(tc.ImageName, bRes)
```

Comment:
can we add a test case where tc.ImageName is not part of the build result?

Reply:
Done!